src/docs/verbose-messaging.dox

```diff
@@ -0,0 +1,62 @@
+@row{\c WARNING, -2, Warning conditions potentially signaling non-imminent errors and behaviors. Important to be aware of and useful for diagnosing potential error states. }
+@row{\c NOTICE, -1, Messages for significant events in WiredTiger\, usually worth noting. Notice messages are associated with normal runtime operations i.e not produced from any state considered harmful or error-prone.}
+@row{\c INFO, 0, Informational style messages\, usually describing statuses and actions performed by WiredTiger. }
+@row{\c DEBUG, 1, Low severity messages\, useful for debugging purposes. }
+</table>
+
+@section verbosity_categories Verbosity Categories
+
+Each verbose message is associated with a category. Verbose categories represent a range of different
+sub-systems in WiredTiger, where messages will denote to a specific operation or event under a given sub-system.
+To see the complete exhaustive list of available verbose categories, refer to the \c 'verbose' configuration option
+under ::wiredtiger_open.
+
+Each verbosity category can be individually configured with a verbosity level (as described in @ref verbosity_levels).
+This allowing the end user to fine tune and tailor the verbosity of each category to fit their use case
+e.g configuring WiredTiger's runtime for debugging/monitoring a specific sub-system.
+
+When a category is assigned a verbosity level, any associated message less than or equal to (\c <=) that level will be displayed.
+As an example, configuring the \c checkpoint category with the \c DEBUG verbosity level would allow any \c checkpoint related messages
+less than or equal to the \c DEBUG level be displayed.
+Any category that is not explicitly configured with a verbosity level will default to the \c NOTICE (\c -1) level, effectively masking
+\c INFO and \c DEBUG informational messages.
+
+@subsection verbosity_configuration Configuring Verbose Categories
+
+Configuration of the verbose messaging system is achieved via the \c verbose configuration option when either
+opening a wiredtiger connection (::wiredtiger_open) or re-configuring a wiredtiger connection (WT_CONNECTION::reconfigure).
+The \c verbose configuration takes in a list of categories, where each category can be optionally associated with a verbosity level.
+An example configuring a WiredTiger connection with verbosity enabled:
+
+@snippet ex_verbose.c Configure verbose_messaging
+
+In the above example:
+- \c api is configured to the \c DEBUG verbosity level.
+- \c version is configured to the \c DEBUG verbosity level. If a category is passed without a verbosity
+level (.e.g \c <code>:0</code>), the category will default to the \c DEBUG level (i.e \c <code>verbose=[version]</code> is equivalent to <code>verbose=[version:1]</code>).
+- \c write is configured to the \c INFO verbosity level.
+
+When configuring verbosity levels, the lowest value the user can associate with a category is \c 0 (\c INFO). This in effect ensuring
```

Comment:
I would say "This in effect ensures" or "This ensuring"

Reply:
Fixed :) 6c1bc279217